dev/ScrollBar/ScrollBar_themeresources.xaml

```diff
@@ -182,22 +185,22 @@
 
     <x:String x:Key="ScrollBarContractDelay">00:00:02</x:String>
     <x:String x:Key="ScrollBarContractFinalKeyframe">00:00:02.1</x:String>
-    <x:Double x:Key="ScrollBarSize">16</x:Double>
+    <x:Double x:Key="ScrollBarSize">12</x:Double>
+    <Thickness x:Key="ScrollBarMargin">1</Thickness>
 
-    <x:Double x:Key="ScrollBarVerticalThumbMinHeight">24</x:Double>
-    <x:Double x:Key="ScrollBarVerticalThumbMinWidth">2</x:Double>
-    <x:Double x:Key="ScrollBarVerticalThumbMaxWidth">6</x:Double>
+    <x:Double x:Key="ScrollBarVerticalThumbMinHeight">30</x:Double>
+    <x:Double x:Key="ScrollBarVerticalThumbMinWidth">8</x:Double>
 
-    <x:Double x:Key="ScrollBarHorizontalThumbMinWidth">24</x:Double>
-    <x:Double x:Key="ScrollBarHorizontalThumbMinHeight">2</x:Double>
-    <x:Double x:Key="ScrollBarHorizontalThumbMaxHeight">6</x:Double>
+    <x:Double x:Key="ScrollBarHorizontalThumbMinWidth">30</x:Double>
+    <x:Double x:Key="ScrollBarHorizontalThumbMinHeight">8</x:Double>
+
+    <Thickness x:Key="ScrollBarThumbBorderThickness">6,6,6,6</Thickness>
 
     <x:Double x:Key="ScrollBarButtonArrowIconFontSize">8</x:Double>
     <x:String x:Key="ScrollBarExpandBeginTime">00:00:00.40</x:String>
     <x:String x:Key="ScrollBarContractBeginTime">00:00:02.00</x:String>
     <CornerRadius x:Key="ScrollBarCornerRadius">3,3,3,3</CornerRadius>
-    <CornerRadius x:Key="ScrollBarTrackCornerRadius">8,8,8,8</CornerRadius>
-
+    <CornerRadius x:Key="ScrollBarTrackCornerRadius">6,6,6,6</CornerRadius>
```

Comment:
`<CornerRadius x:Key="ScrollBarTrackCornerRadius">6</CornerRadius>`  works too.  

(same for ScrollBarCornerRadius)